test/wallet/lockscreen.component.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { expect } from 'chai'
+import { spy } from 'sinon'
+import LockScreen from '../../plugins/Wallet/js/components/lockscreen.js'
+import PasswordPrompt from '../../plugins/Wallet/js/containers/passwordprompt.js'
+import NewWalletButton from '../../plugins/Wallet/js/containers/newwalletbutton.js'
+
+const lockedEncryptedScreen = shallow(<LockScreen unlocked={false} encrypted={true} unlocking={false} />)
+const lockedUnencryptedScreen = shallow(<LockScreen unlocked={false} encrypted={false} unlocking={false} />)
+const unlockedEncryptedScreen = shallow(<LockScreen unlocked={true} encrypted={true} unlocking={false} />)
+const lockedEncryptedUnlockingScreen = shallow(<LockScreen unlocked={false} encrypted={true} unlocking={true} />)
+
+describe('wallet lock screen component', () => {
+	it('renders an empty div if the wallet is unlocked and encrypted and not unlocking', () => {
```

Comment:
I've noticed that some descriptions are of the form `it('renders')`, while others are `it('should render')`. Should we standardize?